jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentials.java

```diff
@@ -82,11 +96,32 @@ RegistryCredentials retrieve(@Nullable String registry) throws MojoExecutionExce
       if (result.getServer() != null) {
         registryServer = result.getServer();
       }
+    } else if (isEncrypted(registryServer.getPassword())) {
+      mavenBuildLogger.warn(
+          "Server password for registry "
+              + registry
+              + " appears to be encrypted, but there is no decrypter available");
     }
 
     return new RegistryCredentials(
         CREDENTIAL_SOURCE,
         Authorizations.withBasicCredentials(
             registryServer.getUsername(), registryServer.getPassword()));
   }
+
+  // pattern cribbed directly from
+  // https://github.com/sonatype/plexus-cipher/blob/master/src/main/java/org/sonatype/plexus/components/cipher/DefaultPlexusCipher.java
+  private static final Pattern ENCRYPTED_STRING_PATTERN =
```

Comment:
In this codebase, we're following a convention of organizing classes in the order:

1. static->non-static
1. members->inner class->constructor->methods
1. public->private
1. final->nonfinal

So a `ENCRYPTED_STRING_PATTERN` would be under `CREDENTIAL_SOURCE`, `isEncrypted` would be under `ENCRYPTED_STRING_PATTERN`.